pkg/node/node.go

```diff
@@ -307,6 +311,21 @@ func NewBee(addr string, logger logging.Logger, o Options) (*Bee, error) {
 
 	b.pullerCloser = puller
 
+	// Initiate the MultiResolver and all ENS resolvers.
+	// TODO: consider refactoring into resolver.Service that handles connection
+	// logic and logging.
+	multiResolver := resolver.NewMultiResolver()
+	for idx, ep := range o.ENSResolverEndpoints {
+		ensR := ens.NewClient()
+		if err := ensR.Connect(ep); err != nil {
+			logger.Warningf("ens resolver [%d]: failed to connect: %v", idx, err)
+		} else {
+			logger.Infof("ens resolver [%d]: connected to endpoint %q", idx, ep)
+			_ = multiResolver.PushResolver(".eth", ensR)
```

Comment:
This will able to resolve only .eth domains, not .test, for example. Swarm `--ens-api` flag provides possibilities to specify default, or any tld resolvers with their specific contract addresses.